platformproxy/VncProxyPlugin.cpp

```diff
@@ -11,6 +11,7 @@ namespace
         Q_PLUGIN_METADATA( IID QPlatformIntegrationFactoryInterface_iid FILE "metadata.json" )
 
       public:
+        using QPlatformIntegrationPlugin::create;
```

Comment:
What is this statement good for ?

Reply:
I was getting error: ‘virtual QPlatformIntegration* QPlatformIntegrationPlugin::create(const QString&, const QStringList&)’ was hidden [-Werror=overloaded-virtual]

Reply:
Guess the create without arc/argv seems to be an outdated API that is kept for compatibility reasons.
However - from the technical point of view it is legal to use the outdated API and in the current implementation it would bypass the initialization. 

So IMHO the warning is actually relevant and should be resolved instead of being suppressed by the using statement.

Reply:
Implementation of the outdated API added ( completely untested )